process/range-input-generator.py

```diff
@@ -0,0 +1,66 @@
+"""
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import csv
+import os
+from itertools import combinations 
+
+# Print iterations progress
+def printProgressBar (iteration, total, progress="Progress", length = 100, fill = '█', printEnd = "\r"):
```

Comment:
As much as I personally prefer camel case, snake case is the standard for Python. https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

Reply:
changed